src/operations.cc

```diff
@@ -221,10 +221,8 @@ namespace sharp {
 
     VipsImage *vips_matrix = kernel.get_image();
 
-    int i = 0;
-    for(int k : kernel_v) {
```

Comment:
C++11 range based loops are [used elsewhere](https://github.com/lovell/sharp/blob/master/src/utilities.cc#L141) so this should work. What was the error?